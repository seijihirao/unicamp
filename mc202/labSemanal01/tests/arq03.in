10
0.237101 
0.919107 0.933732 
0.086825 0.621519 0.947153 
0.493106 0.004131 0.425407 0.453898 
0.184390 0.494412 0.127104 0.974149 0.557015 
0.211105 0.806732 0.040117 0.940512 0.396283 0.841954 
0.222168 0.719003 0.264434 0.238189 0.317051 0.168653 0.879532 
0.422037 0.733863 0.628881 0.773168 0.075179 0.176077 0.940226 0.577722 
0.915795 0.117241 0.026190 0.779207 0.100007 0.686957 0.907721 0.564418 0.420171 
0.308809 0.243249 0.484665 0.895262 0.334119 0.233101 0.617668 0.711731 0.783695 0.116585 
