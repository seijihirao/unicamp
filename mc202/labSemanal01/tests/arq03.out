-0.841704699701 
1.407382511573 1.455612146598 
-1.337277802814 0.426010844713 1.499871287225 
0.002536511553 -1.609982174695 -0.220718056759 -0.126761781101 
-1.015532548713 0.006843376738 -1.204447616887 1.588897422414 0.213292596668 
-0.927433082068 1.036797538176 -1.491309241634 1.477970910733 -0.316761809954 1.152950988080 
-0.890950042595 0.747488980835 -0.751567221811 -0.838116744631 -0.578049361564 -1.067429284877 1.276873943634 
-0.231831483770 0.796493587773 0.450288900990 0.926111762451 -1.375683432182 -1.042946767925 1.477027753426 0.281579164358 
1.396460354226 -1.236973352973 -1.537237045190 0.946026891744 -1.293806825111 0.641809193894 1.369834297939 0.237705860811 -0.237985090536 
-0.605229440325 -0.821430115350 -0.025299820020 1.328747595527 -0.521763316395 -0.854895710990 0.413311198595 0.723507722312 0.960827206410 -1.239136678824 

0.492336836364 0.303236796057 
